README.md

```diff
@@ -33,8 +33,8 @@ context after the span finished.
 # Example
 
 ```rust
-use skywalking_rust::context::trace_context::TracingContext;
-use skywalking_rust::reporter::grpc::Reporter;
+use skywalking::context::trace_context::TracingContext;
+use skywalking::reporter::grpc::Reporter;
```

Comment:
The compiling doc in the readme is still skywalking-rust, I think.

Reply:
Compiling doc is means the doc generated by `cargo doc`? With 0.2.0 released, the `skywalking` will be a new crate and the name is `skywalking` in docs.rs.

Reply:
Oh, I didn't mean this is wrong. I mean https://github.com/apache/skywalking-rust#how-to-compile doesn't match here.